x-pack/plugins/infra/public/pages/metrics/inventory_view/components/node_details/tabs/metrics/metrics.tsx

```diff
@@ -291,167 +327,113 @@ const TabComponent = (props: TabProps) => {
 
       <EuiFlexGrid columns={2} gutterSize={'l'} responsive={false}>
         <EuiFlexItem>
-          <ChartHeader title={CPU_CHART_TITLE} metrics={cpuChartMetrics} />
-          <Chart ref={cpuChartRef} size={CHART_SIZE}>
-            <MetricExplorerSeriesChart
-              type={MetricsExplorerChartType.line}
-              metric={cpuChartMetrics[0]}
-              id={'0'}
-              series={systemMetricsTs!}
-              stack={false}
-            />
-            <MetricExplorerSeriesChart
-              type={MetricsExplorerChartType.line}
-              metric={cpuChartMetrics[1]}
-              id={'0'}
-              series={userMetricsTs}
-              stack={false}
-            />
-            <Axis
-              id={'timestamp'}
-              position={Position.Bottom}
-              showOverlappingTicks={true}
-              tickFormat={formatter}
-            />
-            <Axis
-              id={'values'}
-              position={Position.Left}
-              tickFormat={cpuFormatter}
-              domain={getDomain(cpuTimeseries, cpuChartMetrics)}
-              ticks={6}
-              showGridLines
-            />
-            <Settings
-              onPointerUpdate={pointerUpdate}
-              tooltip={tooltipProps}
-              theme={getTimelineChartTheme(isDarkMode)}
-            />
-          </Chart>
+          <ChartSection
+            title={CPU_CHART_TITLE}
+            style={MetricsExplorerChartType.line}
```

Comment:
All the charts are line chart types, so maybe we could also remove the style as a prop?

Reply:
We have plans to make one of them an area chart, so it probably makes sense to keep it for now.